.github/workflows/meson.yml

```diff
@@ -0,0 +1,52 @@
+name: meson checks
```

Comment:
[suggestion] Could you follow the same naming schema we already have for the pipelines? For example, add the suffix `on_PR_` if you are thinking about running this pipeline on PRs

Reply:
Done

Reply:
Actually, I was referring to the filename 😅 . Apart from that the rest of the PR looks great and I tried to use meson to configure and compile the project and it worked like a charm 👏

Reply:
Renamed